src/storage/txn/process.rs

```diff
@@ -262,6 +274,21 @@ impl<E: Engine, S: MsgScheduler> Executor<E, S> {
         let scheduler = self.take_scheduler();
         let waiter_mgr_scheduler = self.take_waiter_mgr_scheduler();
         let detector_scheduler = self.take_detector_scheduler();
+        let label = match tag{
```

Comment:
this is verbose, `SCHEDULER_ROCKSDB_PERF_CONTEXT_HISTOGRAM_VEC.get(tag)` not work?